tests/integration/test_ssl_cert_authentication/test.py

```diff
@@ -1,8 +1,10 @@
+import json
 import logging
 import os.path
 import ssl
 import urllib.parse
 import urllib.request
+import time
```

Comment:
Looks like `time` is imported but not used anywhere. Unrelated to this PR, we can also remove `logging` as it's also not used in this test file.